client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -148,13 +148,13 @@ class SharingServiceDescription extends Component {
 			},
 			p2_github: function () {
 				if ( this.props.numberOfConnections > 0 ) {
-					return this.props.translate( 'Workspace connected to Github.', {
-						comment: 'Embed Github Issues in P2 posts.',
+					return this.props.translate( 'Workspace connected to GitHub.', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.